backend/sse/src/server.ts

```
@@ -0,0 +1,197 @@
+const authenticate = async (authorization: string) => {
+    try {
+        const authResponse = await fetch(`${process.env.DJANGO_BACKEND_URL}/api/v1/auth/whoami`, {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json",
+                "Authorization": `Bearer ${authorization}`,
+            }
+        });
+        return authResponse;
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)